backend/src/utils/db.py

```
@@ -0,0 +1,271 @@
+        VALUES (?, ?, ?, ?, ?, ?, ?)
+        ''', (
+            position_data['token_address'],
+            position_data.get('token_symbol'),
+            position_data['buy_price'],
+            position_data.get('highest_price', position_data['buy_price']),
+            position_data['amount_tokens'],
+            position_data.get('buy_amount_sol'),
+            position_data.get('purchase_time', datetime.now().isoformat())
+        ))
+        conn.commit()
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)